lib/ts/recipe/session/recipe.ts

```diff
@@ -79,39 +78,17 @@ export default class SessionRecipe extends RecipeModule {
                 issuer: this.config.jwt.issuer,
                 override: this.config.override.openIdFeature,
             });
-
```

Comment:
we will always need the openIdRecipe now cause we will / should always expose the jwks endpoint now?

Reply:
We don't always need to expose the JWKS endpoint from the backend. The FE doesn't need it, and the m2m/multiple backends use-case is fairly rare, I think.

Reply:
Yea.. but we should i think. It's expected and easier docs + less configuration for the user. The m2m, and multiple backend is not really rare - most medium / large users have this need.

Reply:
We also decided to getting rid of `jwt: {enable: true}`, and to initlaise openid inside session anyway